GUI/GUIListPanel.cpp

```diff
@@ -437,10 +437,12 @@ void GUIListPanel::OnMouseDown(int X, int Y, int Buttons, int Modifier) {
 void GUIListPanel::OnMouseWheelChange(int x, int y, int modifier, int mouseWheelChange) {
 	if (!m_MouseScroll) {
 		return;
-	} else if (m_VertScroll->_GetVisible() && m_VertScroll->PointInside(x, y)) {
+	}
+	else if ((PointInsideList(x, y) && !m_MultiSelect) || (m_VertScroll->_GetVisible() && m_VertScroll->PointInside(x, y)) =) {
 		ScrollBarScrolling(mouseWheelChange);
-	} else if (PointInsideList(x, y) && !m_MultiSelect) {
-		SelectionListScrolling(mouseWheelChange);
+	}
+	if (m_HotTracking && GetItem(x, y) != nullptr && (GetItem(x, y) != GetSelected())) {
```

Comment:
Minor thing, but add a space above this if and the closing bracket above to visually separate it